main.go

```diff
@@ -17,16 +17,18 @@ import (
 
 // Config contains the application configuration parameters
 type Config struct {
-	BaseDir      string   `envconfig:"BASE_DIR" default:"."`
-	Port         string   `envconfig:"PORT" default:"8000"`
-	AwsRegion    string   `envconfig:"AWS_REGION" default:"us-west-1"`
-	S3Bucket     string   `envconfig:"S3_BUCKET" default:"nitro-junk"`
-	ClusterSeeds []string `envconfig:"CLUSTER_SEEDS" default:"127.0.0.1"`
-	CacheSize    int      `envconfig:"CACHE_SIZE" default:"512"`
-	RedisPort    int      `envconfig:"REDIS_PORT" default:"6379"`
-	ClusterName  string   `envconfig:"CLUSTER_NAME" default:"default"`
+	BaseDir                 string   `envconfig:"BASE_DIR" default:"."`
+	Port                    string   `envconfig:"PORT" default:"8000"`
+	AwsRegion               string   `envconfig:"AWS_REGION" default:"us-west-1"`
+	S3Bucket                string   `envconfig:"S3_BUCKET" default:"nitro-junk"`
+	ClusterSeeds            []string `envconfig:"CLUSTER_SEEDS" default:"127.0.0.1"`
+	CacheSize               int      `envconfig:"CACHE_SIZE" default:"512"`
+	RedisPort               int      `envconfig:"REDIS_PORT" default:"6379"`
+	ClusterName             string   `envconfig:"CLUSTER_NAME" default:"default"`
+	MemberlistAdvertiseAddr string   `envconfig:"MEMBERLIST_ADVERTISE_ADDRESS"`
```

Comment:
Let's not manipulate the names. Just make it `MEMBERLIST_ADVERTISE_ADDR`. It reduces chances of getting it wrong if they match the struct fields exactly.